### DISCUSSION TITLE
Hey ninjamuffin. quick question

### DISCUSSION THREAD

**Discussion by AbnormalPoof** [FunkinCrew/Funkin#1728] [Q&A] [answered]
why did you fork haxeflixel
![Screenshot 2021-09-05 10 34 05 PM](https://user-images.githubusercontent.com/86753001/132157142-009b0e71-c021-43fe-957c-9a7bcb4a4918.png)

**Comment by Trayfellow** [comment, marked as answer]
probably for pull requests stuff like this

![funnehehiaimage](https://user-images.githubusercontent.com/48515908/132300669-43f0638b-d2d4-4679-95db-3a09636513c0.PNG)

**Reply by AbnormalPoof** [reply]
makes sense

**Reply by ninjamuffin99** [reply]
yep

**Comment by Cheemsandfriends** [comment]
When you make PR's you automatically make a fork of the thing. So that you can modify the files and propose the stuff
